src/pymoca/ast.py

```diff
@@ -895,6 +901,18 @@ def remove_initial_equation(self, e: Equation) -> None:
         """
         self.initial_equations.remove(e)
 
+    def extend(self, other: "Class") -> None:
```

Comment:
I see I named it `_extend` in the past, but it's more like "_merge_tree" or something right? I got confused a little, because "extends" has a specific meaning in Modelica, and `Class` also has an `extends` property that is very different from `def extend`.

Reply:
I agree renaming would be good.

Reply:
Since I moved it back to the place from whence it came (your 7 year old code), I think I'm going to pass on renaming for now and avoid complicating the diffs.